### DISCUSSION TITLE
Renderer in separate window

### DISCUSSION THREAD

**Discussion by bejucoo** [cables-gl/cables_docs#130] [Answers & Questions] [answered]
Hello, I wanted to know if it's possible to get the output of the patch in a separate window. I saw [this ](https://forum.cables.gl/t/detached-preview-window-kind-of-solved/911) and [this ](https://forum.cables.gl/t/open-renderer-in-new-window-is-not-working/662) discussions in the previous forum but I can´t find the 'open renderer in new window' option. Also, I'm not aware if the feature has been mentioned in a changelog or a blog post.

**Comment by steam0r** [comment, marked as answer]
we are currently reworking the "remote viewer" feature on dev, you can try it, if you want:

![Bildschirmfoto 2022-01-28 um 14 16 07](https://user-images.githubusercontent.com/151271/151553241-64b2d8e1-b544-4403-9edf-70dc33a18216.png)

let us know if that suits your needs

**Reply by bejucoo** [reply]
Yes, it works for what I need. Thank you!
